src/components/visualizations/chart/highcharts/helpers.ts

```diff
@@ -136,18 +138,39 @@ function getExtremeOnAxis(min: number, max: number) {
     return { axisMin, axisMax };
 }
 
-export function shouldFollowPointer(chartOptions: any) {
-    const yMax = parseFloat(get(chartOptions, 'yAxisProps.max', ''));
-    const yMin = parseFloat(get(chartOptions, 'yAxisProps.min', ''));
+function shouldFollowPointerForDualAxes(chartOptions: any) {
+    const yAxes = get(chartOptions, 'yAxes', []);
+    if (yAxes.length <= 1) {
```

Comment:
return yAxes.length > 1; should work too

Reply:
Yes, it should work in case min/max setting is configured
I did it in second `return`.